internal/gengapic/testdata/empty_client_init.want

```diff
@@ -41,37 +58,40 @@ func NewClient(ctx context.Context, opts ...option.ClientOption) (*Client, error
 	if err != nil {
 		return nil, err
 	}
-	c := &Client{
+	client := Client{CallOptions: defaultCallOptions()}
```

Comment:
This appears to have type `Client`, but the return type is `*Client`. Should this line be `&Client`?

Reply:
Yes, fixed